src/index.js

```diff
@@ -93,7 +93,9 @@ yargs // eslint-disable-line no-unused-expressions
     aliases: ['$0'],
     desc: 'Launch interactive mode',
     handler: async options => {
-      await checkGuidelines(options)
+      const code = await checkGuidelines(options)
+      console.log()
```

Comment:
Is this console.log a mistake or is it to flush stdout ? If the later, at least add a comment.

Reply:
It's a mistake